client/src/components/TrailsPage.js

```diff
@@ -57,19 +59,30 @@ const TrailsPage = (props) => {
   }
   const trailListItems = trails.map((trailItem) => {
     return (
-      <li key={trailItem.id}>
-        <Link to={`/trails/${trailItem.id}`}>{trailItem.name}</Link>
-      </li>
+      <div>
+        <li key={trailItem.id}>
+          <Link to={`/trails/${trailItem.id}`}>{trailItem.name}</Link>
+        </li>
+      </div>
     )
   })
 
   return (
-    <div>
-      <h1>All the amazing trails in Massachusetts</h1>
-      <ErrorList errors={errors} />
-      <TrailForm addTrail={addTrail} />
-      <ul>{trailListItems}</ul>
+  <div className="trail-bg-img-trails">       
+      <div className="trails-and-form">
+        <div className="trails-left">  
+        <h3>All the amazing trails in Massachusetts:</h3>  
+          <ul>{trailListItems}</ul>
+        </div>
+        <div>
+        </div>
+        <div className="form-right">
+          <h3>Add New trail:</h3>
+          <ErrorList errors={errors} />
+          <TrailForm addTrail={addTrail} />
+        </div>
     </div>
+  </div>
```

Comment:
This section in the `return` should all be indented one so that it is nested under the return

Reply:
I think it's better now.